Library/Homebrew/cask/audit.rb

```diff
@@ -754,7 +754,7 @@ def cask_plist_min_os
     def audit_github_prerelease_version
       odebug "Auditing GitHub prerelease"
       user, repo = get_repo_data(%r{https?://github\.com/([^/]+)/([^/]+)/?.*}) if online?
-      return if user.nil?
+      return if user.nil? || repo.nil?
```

Comment:
Might be worth extracting the boolean into a method given the number of times it's used here.

Reply:
I wanted to, but Sorbet doesn't like that?

Reply:
Boo

Reply:
`get_repo_data` should technically return `T.nilable([String, String])` but even then Sorbet's Tuple support isn't intelligent enough to figure out you only need to check nil once here.

Reply:
I guess it might work with `T.nilable([String, String])` and

```rb
repo_data = ...
return if repo_data.nil?

user, repo = repo_data
```

Not sure if that necessarily reads any better though.